widgets/src/main/java/ru/startandroid/widgets/weatherwidget/WeatherWidgetDbDataHelper.kt

```diff
@@ -18,12 +19,16 @@ class WeatherWidgetDbDataHelper @Inject constructor() : WidgetDbDataHelper {
 
     override fun refreshData(config: WidgetConfigEntity?): WidgetData? {
         val api: WeatherAPI = WeatherAPI.create()
```

Comment:
Let's inject it in WeatherWidgetDbDataHelper constructor instead of manual creating.